weather.py

```diff
@@ -24,7 +24,9 @@ def convert_date(iso_string):
     Returns:
         A date formatted like: Weekday Date Month Year e.g. Tuesday 06 July 2021
     """
-    pass
+    date = datetime.fromisoformat(iso_string)
+  
+    return(date.strftime("%A %d %B %Y"))
```

Comment:
Minor syntax comment: `return` is a special statement and you don't need to encase values in `()`. It helps to keep a space between `return` and the value being returned, because `return()` can look like a function which makes code a little less readable.